.github/actions/setup-sentry/action.yml

```diff
@@ -162,7 +162,7 @@ runs:
             --name sentry_zookeeper \
             -d --network host \
             -e ZOOKEEPER_CLIENT_PORT=2181 \
-            confluentinc/cp-zookeeper:4.1.0
```

Comment:
It seems the CI was running an older version.

Reply:
What version do we run in production?

Reply:
I believe we run [4.1.2-3](https://github.com/getsentry/ops/blob/c823e62f930ecc6c97bb08898c71e49edc7232f6/cookbooks/getsentry/attributes/default.rb#L643) and version [3.4.11](https://github.com/getsentry/ops/blob/c823e62f930ecc6c97bb08898c71e49edc7232f6/cookbooks/getsentry/attributes/default.rb#L631) for zookeeper.

Should I make the CI test against those versions?

As curiosity, we run a range of versions across the org: https://github.com/search?q=org%3Agetsentry+cp-kafka&type=code

Reply:
Yeah I think we should have CI match production, let's leave comments here

Reply:
@billyvg I've discovered that version of zookeeper we use in production is no longer available on Docker hub, thus, we cannot match the versions between CI and production.

FYI, this part of the PR is completely unnecessary to me.

Reply:
Asked about the versions picked in https://github.com/getsentry/sentry/pull/20543#pullrequestreview-755065322